solidus_sitemap.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.executables = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency 'sitemap_generator', '~> 6.0.1'
+  s.add_dependency 'sitemap_generator', '~> 6.1'
```

Comment:
Why not just 6.0? We are forcing users to update this dependency if they were on 6.0.x. Is this intentional?

Reply:
Right, no reason to force that update 👍